sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Release History
 
-## 2.0.0b1 (Unreleased)
+## 2.0.0b1 (2020-09-08)
 
-  - Placeholder - NEEDS TO BE CHANGED
+  - Version (2.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure EventGrid.
+  
+  For more information about this, and preview releases of other Azure SDK libraries, please visit https://azure.github.io/azure-sdk/releases/latest/python.html.
+
+  **Breaking Changes**
```

Comment:
So looking at what you put under here, I'm not sure it's set up right.  I'd probably put these under **Features** (and put that header above your top-level preview effort snippet), and if you wanted a breaking changes section (which I thought was reasonable) you could mention the deprecation of the track-1 publishing method?